src/modules/fundingManager/bondingCurveFundingManager/BancorVirtualSupplyBondingCurveFundingManager.sol

```diff
@@ -454,7 +460,7 @@ contract BancorVirtualSupplyBondingCurveFundingManager is
         // An input verification is needed here since the Bancor formula, which determines the
         // issucance price, utilizes PPM for its computations. This leads to a precision loss
         // that's too significant to be acceptable for tokens with fewer than 7 decimals.
-        if (_decimals < 7) {
+        if (_decimals < 7 || _decimals > collateralTokenDecimals) {
```

Comment:
Is this gonna be a problem for USDC and Tether, as they are both 6 decimal tokens?

Reply:
You are of course completely right, this is what I get for rushing to quickly update the setter before the dev call :laughing: It should be as mentioned in the interface. I pushed the fix

Reply:
Haha, all good! 😂 
So the bonding curve (issuance) token decimals need to be at least 7 (for precision) and also need to be at least equal to the collateral tokens decimals, not lower, correct? That's how I understand the fix you pushed now, which looks good to me, just making sure.

Initially, I thought `_decimals` was for the collateral token, but it's for the issuance token, I think I get it now. Is there a good reason why someone would set the issuance tokens decimals to anything but 18?

Reply:
Yes, exactly! 
Personally, I can't think of a good reason to not have 18 decimals. But if we are going to "plug in" our bonding curve to an external token maybe that token had a good reason for it? So better to keep it open...